packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -166,7 +168,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 			className,
 			props.className,
 			wrapperProps.className,
-			defaultClassName
+			getBlockDefaultClassName( name )
```

Comment:
This has a regex, so best not to run the for every block selector, which runs more often than render.